erpnext/patches/v13_0/set_values_in_production_plan_items.py

```diff
@@ -0,0 +1,12 @@
+import frappe
+from erpnext.manufacturing.doctype.production_plan.production_plan import get_item_data
+
+def execute():
+    frappe.reload_doctype("Production Plan Item")
+    production_plans = frappe.get_list("Production Plan Item", fields=["name", "item_code"])
+    for production_plan in production_plans:
+        data = get_item_data(production_plan.item_code)
+        frappe.db.set_value("Production Plan Item", production_plan.name, "raw_material_cost", data.raw_material_cost)
```

Comment:
I got the following error:

`AttributeError: 'dict' object has no attribute 'raw_material_cost'`